src/main/java/frc/robot/subsystems/Dashboard.java

```diff
@@ -0,0 +1,51 @@
+package frc.robot.subsystems;
```

Comment:
All of the entries you've added are eventually going to need to hook into their respective subsystems to update settings. We'll either need to monitor this in the periodic function (in which case all of these entries need to be member variables and defined above the constructor, not inside it), or set up callbacks for each entry (a callback essentially says "when the value of this network table entry changes, call this function automatically").

I think we'll probably want callbacks when possible. We have an example of how to do this in DashboardButton.java file, inside the `whenPressed` function.

To register the callback, though, you need a reference to a Network Table entry (they use the `GenericEntry` class). All of the entries you've created so far are widgets, not entries. You can get the entry for a widget by calling `.getEntry()` at the end of what you've written so far. You do this on line 32 right now.